src/App.tsx

```diff
@@ -1,11 +1,18 @@
-import * as React from 'react';
-import Hero from "./components/hero/Hero";
+import React from 'react';
+import Hero from './components/hero/Hero';
+import { ThemeProvider } from 'styled-components';
+import { theme } from "./styles/theme";
+import { GlobalStyles } from "./styles/globalStyles";
+
 
 export const App = () => {
   return (
-    <div>
-      <Hero/>
-    </div>
+    <>
+      <GlobalStyles/>
+      <ThemeProvider theme={theme}>
+        <Hero/>
```

Comment:
Component name 'Hero' might be somewhat misleading as by 'hero' it is often understood the oversized banner on the landing page, whereas in your case it is, basically, the wrapper for your entire app.